dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/versioning/VersionManager.java

```diff
@@ -83,6 +83,7 @@ public static FlowResult processCreateNewVersion(Context context, UUID itemID, S
             context.abort();
             throw new RuntimeException(ex);
         }
+        context.complete();
```

Comment:
@samuelcambien : I see now. Thanks for the clarification.

Looking closer, `versioning.js` has that same sort of `getDSContext.complete()` line before redirects elsewhere in the file. Here's an example: https://github.com/DSpace/DSpace/blob/dspace-6_x/dspace-xmlui/src/main/resources/aspects/Versioning/versioning.js#L168

So, yes, I agree with your suggestion. This PR should be reworked to simply add a `getDSContext.complete()` just before the `doCreateNewVersion()` redirect in versioning.js